lib.py

```diff
@@ -926,6 +926,33 @@ def check_kill_count(client, monster_name, kill_amount, label_jump, label_skip):
     else:
         print('[Action] Could not find Quest Tracker window')
 
+#check how many pirates you killed to do the task
```

Comment:
Thanks. I will test it and it should be on next update.

Reply:
when is the next update?

Reply:
Already updated